src/common/ecmdDefines.H

```diff
@@ -103,8 +103,6 @@
 //#define ECMD_REMOVE_MEMORY_FUNCTIONS
 //#define CIP_REMOVE_MEMORY_FUNCTIONS
 //#define ECMD_REMOVE_INIT_FUNCTIONS
-//#define CIP_REMOVE_INSTRUCTION_FUNCTIONS
-//#define ECMD_REMOVE_PROCESSOR_FUNCTIONS
 #endif // EDBG_BUILD_EBMC
```

Comment:
There should be no changes to this file.  The two commented out lines should be in the file.

Reply:
done